src/Microsoft.Graph.Cli.Core/Microsoft.Graph.Cli.Core.csproj

```diff
@@ -5,6 +5,8 @@
     <RootNamespace>Microsoft.Graph.Cli.Core</RootNamespace>
     <ImplicitUsings>disable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
```

Comment:
Do you also want to add stuff like the Release Notes and Description?

You can check out this file for reference? 
https://github.com/microsoft/kiota-serialization-text-dotnet/blob/f150fc88eaa0d072c09ef0d1a3de47299a0210fa/src/Microsoft.Kiota.Serialization.Text.csproj#L4